main.go

```diff
@@ -1,5 +1,59 @@
 package main
 
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/config"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/reconciler"
+	"go.uber.org/zap"
+)
 
 func main() {
+	//logger := logr.Logger{}
+	ctx := context.Background()
+	configFilePath := os.Getenv("CONFIG_FILE_PATH")
```

Comment:
Can we make the env var more specific and remove the redundant "PATH" suffix?
```suggestion
	configFilePath := os.Getenv("FLEET_RECONCILER_CONFIG")
```